phpunit.xml.dist

```diff
@@ -8,6 +8,6 @@
     <env name="USER" value="test" force="true"/>
   </php>
   <extensions>
-    <bootstrap class="Zalas\PHPUnit\Globals\AnnotationExtension" />
+    <bootstrap class="Zalas\PHPUnit\Globals\Tests\Stub\CombinedExtension" />
```

Comment:
Nice trick. I'm guessing it's needed as we can't register both Annotation and Attribute extensions one after another, right?

People might need a similar trick to migrate from annotations to attributes. Although, I don't expect anyone using this extension so heavily they need complex migration paths. Just replacing annotations with attributes  in one go should be sufficient for most.

Reply:
Ha! I'm itching to remove annotations asap. I guess the latest when PHPUnit 11 is released.